patches/chrome-browser-ui-views-location_bar-location_bar_view.cc.patch

```diff
@@ -10,15 +10,23 @@ index 61d32b558b7c49fa69989219b660bf2adf8fb90c..238b7d421119e21b15e95a86c789f5a2
    if (should_indent) {
      icon_left += icon_indent;
      text_left += text_indent;
-@@ -802,6 +803,7 @@ void LocationBarView::Layout(PassKey) {
+@@ -778,6 +779,7 @@ void LocationBarView::Layout(PassKey) {
+       }
+       selected_keyword_view_->SetCustomImage(image);
+     }
++  BRAVE_LAYOUT_HANDLE_SEARCH_PROMOTION_BUTTON_VISIBILITY
```

Comment:
nit: `BRAVE_LOCATION_BAR_VIEW_LAYOUT_...` (all defines in this file can be updated).